kie-server-parent/kie-server-tests/kie-server-integ-tests-controller/src/test/java/org/kie/server/integrationtests/controller/KieControllerStartupIntegrationTest.java

```diff
@@ -183,6 +185,7 @@ public void testContainerCreatedAfterStartup() throws Exception {
     }
 
     @Test
+    @Category({UnstableOnJenkinsPrBuilder.class})
```

Comment:
```suggestion
```

for me https://github.com/kiegroup/drools/commit/1730221b533e4b8da9c02f5cb8730c2490a39e7e
resolved the issue

as I taken the steps to reproduce indicated in https://issues.redhat.com/browse/DROOLS-5685

that is:
![image](https://user-images.githubusercontent.com/1699252/94902154-07f57900-0498-11eb-9420-1cc84984b394.png)


I get:

![image](https://user-images.githubusercontent.com/1699252/94902065-e72d2380-0497-11eb-8a01-312d664061d4.png)

Reply:
ok, i will try to trigger tests again, thanks @tarilabs for checking

Reply:
@tarilabs This test is not related to the issue which you reference in your comment. Is there any specific reason for removing the unstable flag here?

Reply:
Sorry @sutaakar it indeed it appears I've been marking over something unrelated. Sorry for noise.

Reply:
(please consider this resolved, as wrong comment 🙏 )